global.json

```diff
@@ -1,18 +1,18 @@
 {
   "sdk": {
-    "version": "7.0.100-preview.7.22377.5",
+    "version": "7.0.100",
     "allowPrerelease": true,
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "7.0.100-preview.7.22377.5",
+    "dotnet": "7.0.100",
     "runtimes": {
       "dotnet": [
         "6.0.3"
       ]
     }
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.22466.3"
+    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.22524.5"
```

Comment:
This does not look correct. `Version.Details.xml` also needs to be updated. see https://github.com/dotnet/format/pull/1725. I'll need to check that automatic arcade updates are still working for this repo.